YaqeenDAL/Model/Model.cs

```diff
@@ -235,5 +235,16 @@ public class CountryState: AuditableEntity
         [ForeignKey("CountryId")]
         public virtual Country Country { get; set; }
     }
+    //university
+    public class University: AuditableEntity
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int UniversityId { get; set; }
+        public string Name { get; set; }
+        public string AlphaCode { get; set; }
```

Comment:
Shouldn't this table have country and state IDs?

Reply:
the problem is the CSV file of universities has only university name and alpha code . if there csv table with country and state id we can change the table

Reply:
I did some search kedah, couldn't find this list anywhere for free (weird!) but here is what I think we can do instead:
* Look at [this repo](https://github.com/Geziwen/world-universities/tree/master) (where that csv came from).
* Notice [that it](https://github.com/Geziwen/world-universities/blob/master/importer/import.js#L37) leverages this website http://univ.cc to query for universities.
* That website does allow drilling down by country and for the US it also allows drilling down by state.
* The code uses a library to get a list of all country codes (to pass to the website to filter), when it comes to the US, it only passes edu which is why all US universities are listed under US with no state code.

I went ahead and [scripted this](https://gist.github.com/EngHabu/a82c87a0637b742c806299c50fcd341e) and generated the list we want here: https://docs.google.com/spreadsheets/d/1LbBdgdutUqoCFbqXC08hLAKuxtsorc_gRzDDiC_6bEA/edit?usp=sharing
* You can combine towns and state columns if you like... you can modify the sheet to match what you want to ingest into the database... you can drop the town id column because that's only relevant to the website I used.